repository-data/webfiles/src/main/resources/site/freemarker/common/macro/sections/tableauLookupSection.ftl

```diff
@@ -157,6 +162,10 @@
                         var longitude = data["result"]["longitude"];
                         viz${index}Url = encodeURI("${section.url}".split("?")[0] + "?MSOA Code=" + msoa + "&Lat=" + latitude + "&Lon=" + longitude + "&Distance=" + parseInt(viz${index}Elements.distance().value) + "&Postcode=" + postcode + "&:refresh=yes");
 
+                        // build the download link and base it on div being empty
+                        downloadVizLink = encodeURI("${section.url}".split("?")[0]+".csv" + "?MSOA Code=" + msoa + "&Lat=" + latitude + "&Lon=" + longitude + "&Distance=" + parseInt(viz${index}Elements.distance().value) + "&Postcode=" + postcode + "&:refresh=yes");
```

Comment:
Instead of a global variable you could pass this into 'loadViz' as an argument.

Reply:
hi, not sure about this, can you write into the code what you mean. Thanks in advance.

Reply:
Done